src/components/SlateEditor/plugins/embed/AudioPlayerMounter.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2018-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { Fragment, useEffect } from 'react';
+import { injectT } from '@ndla/i18n';
+import { initAudioPlayers } from '@ndla/article-scripts';
+// @ts-ignore
+import { AudioPlayer, FigureCaption, FigureLicenseDialog } from '@ndla/ui';
+import { getLicenseByAbbreviation } from '@ndla/licenses';
+import { Audio, TranslateType } from '../../../../interfaces';
+
+interface Props {
+  t: TranslateType;
+  audio: Audio;
+  speech: boolean;
+}
+
+const AudioPlayerMounter: React.FC<Props> = ({ t, audio, speech }) => {
```

Comment:
Du kan vurdere å sløyfe `React.FC<Props>` og heller skrive det som `const AudioPlayerMounter = ({ t, audio, speech }: Props) => {`

Reply:
Ved å bruke FC så sier han at komponenten er en funksjonel komponent så det kommer med en del props automatisk vs å bare ta inn propsene han har satt. Nå har jeg ikke sett på akkurat koden, men om det f.eks kan bli bruk for children e.l. så er det jo greit å beholde FC